CancelablePromise.spec.js

```diff
@@ -106,10 +106,16 @@ describe(__filename, () => {
     rejectedPromise.cancel();
     rejectedPromise.then(() => {
       hasFailed = true;
-      done(new Error('Callback should not be executed'));
+      done(new Error('Success callback should not be executed'));
```

Comment:
what happens in the other way around:
```javascript
p.then(() => {done()})
p.cancel()
```

maybe with a `setTimeout` to ensure `cancel` is called before the execution of the `then` :wink: